licenses.d2iq.yaml

```diff
@@ -334,22 +334,22 @@ resources:
       - license_path: LICENSE
         ref: ${image_tag}
         url: https://github.com/brancz/kube-rbac-proxy
-  - container_image: ghcr.io/fluxcd/helm-controller:v0.37.4
+  - container_image: ghcr.io/fluxcd/helm-controller:v1.0.1
     sources:
       - license_path: LICENSE
         ref: ${image_tag}
         url: https://github.com/fluxcd/helm-controller
-  - container_image: ghcr.io/mesosphere/dkp-container-images/ghcr.io/fluxcd/kustomize-controller:v1.2.2-d2iq.1
+  - container_image: ghcr.io/fluxcd/kustomize-controller:v1.3.0
     sources:
       - license_path: LICENSE
         ref: ${image_tag%-d2iq.1}
         url: https://github.com/fluxcd/kustomize-controller
-  - container_image: ghcr.io/fluxcd/notification-controller:v1.2.4
+  - container_image: ghcr.io/fluxcd/notification-controller:v1.3.0
     sources:
       - license_path: LICENSE
         ref: ${image_tag}
         url: https://github.com/fluxcd/notification-controller
-  - container_image: ghcr.io/fluxcd/source-controller:v1.2.4
+  - container_image: ghcr.io/fluxcd/source-controller:v1.3.0
```

Comment:
1.4.1 is latest. any reason we want to keep it to 1.3.0? https://github.com/fluxcd/source-controller/releases

Reply:
same as other comments, trying to keep things in scope given limited time left for 2.13 release.